src/mds/api/v1/api.py

```diff
@@ -574,3 +574,144 @@ def convert_binary(binary):
     result, message = FFmpeg().convert(binary, type, extension)
     logging.debug('FFmpeg: %s %s' % (result, message))
     return result
+
+def get_binary_checksum(procedure_guid, element_id, file_size):
+    """Returns checksums of the BinaryResource object associated with an encounter
+
+    If binary data is empty (no chuncks have been transferred), it fills data
```

Comment:
Saving procedure, which must be called before sending chunks, already creates an empty binary file with size 0.